packages/suite-web-landing/pages/index.tsx

```diff
@@ -85,6 +125,103 @@ const Index = () => {
+                                                values={{
+                                                    strong: chunks => <strong>{chunks}</strong>,
+                                                }}
+                                            />
+                                        </CollapsibleBoxContentP>
+                                        <CollapsibleBoxContentP>
+                                            <Translation
+                                                id="TR_SUITE_WEB_LANDING_DEPRECATION_BANNER_PARAGRAPH_1"
+                                                values={{
+                                                    strong: chunks => <strong>{chunks}</strong>,
+                                                }}
+                                            />
+                                        </CollapsibleBoxContentP>
+                                        <CollapsibleBoxContentP>
+                                            <Translation
+                                                id="TR_SUITE_WEB_LANDING_DEPRECATION_BANNER_PARAGRAPH_2"
+                                                values={{
+                                                    strong: chunks => <strong>{chunks}</strong>,
+                                                }}
+                                            />
+                                        </CollapsibleBoxContentP>
+
+                                        <CollapsibleBoxContentHeadline>
+                                            <Translation id="TR_SUITE_WEB_LANDING_DEPRECATION_BANNER_HEADLINE_1" />
+                                        </CollapsibleBoxContentHeadline>
+                                        <CollapsibleBoxContentP>
+                                            <Translation
+                                                id="TR_SUITE_WEB_LANDING_DEPRECATION_BANNER_PARAGRAPH_3"
+                                                values={{
+                                                    strong: chunks => <strong>{chunks}</strong>,
+                                                }}
+                                            />
+                                        </CollapsibleBoxContentP>
+
+                                        <CollapsibleBoxContentHeadline>
+                                            <Translation id="TR_SUITE_WEB_LANDING_DEPRECATION_BANNER_HEADLINE_2" />
+                                        </CollapsibleBoxContentHeadline>
+                                        <CollapsibleBoxContentP>
+                                            We recommend to{' '}
+                                            <StyledCollapsibleBoxLink
+                                                onClick={() =>
+                                                    ref?.current?.parentElement?.parentElement?.parentElement?.parentElement?.parentElement?.scrollTo(
```

Comment:
where are you trying to scroll?

Reply:
to the top section with download button

Reply:
so in other words to the very top of the page? Why not just
```javascript
window.scrollTo({
  top: 0,
  left: 0,
  behavior: 'smooth'
});
```
and ged rid of the `ref`? 

If this is not possible, let's create the `ref` on the correct element and avoid this crazy fragile reference on parent of parent of parent of parent of parent...